docs/description.md

```diff
@@ -0,0 +1,109 @@
+attestation. The Fobnail Token uses default policies during the local
+attestation.
+
+The attestation architecture [[6]](architecture#Attestation) is based on the
+Reference Interaction Model for Challenge-Response-based Remote Attestation
+(CHARRA) [[7]](https://tools.ietf.org/id/draft-birkholz-rats-reference-interaction-model-00.html).
+This is the IEFT attestation standard, which specifies the communication
+between the Attester and the Verifier (Fobnail). The Fobnail provides the
+ability to perform attestation in absence of any connectivity. During the
+attestation development, we will use the proof-of-concept implementation
+[[8]](https://github.com/Fraunhofer-SIT/charra) of the proposed architecture,
+which is developed by the Fraunhofer Institute.
+
+To demponstrate the capabilities of implemented solution,
+we will create the example use case that uses the TrenchBoot - our previous
+NLnet project [[9]](https://nlnet.nl/project/OpenDRTM/), to verify the integrity
+of the system firmware, D-RTM Configuration Environment (DCE), Linux Kernel,
+and initrd.
+
+The main use case that will be presented as a demonstration is boot
+time attestation. The Fobnail Token will be used to verify an operating system
+during boot time. We will run inside DLME
+[[10]](https://trustedcomputinggroup.org/wp-content/uploads/DRTM-Specification-Overview_June2013.pdf)
+(Dynamically Launched Measured Environment) the Zephyr RTOS which will check
+if the measurements of the operating system met the reference measurements
+in the Fobnail Token. If OS is compromised the Fobnail Token warns the user
+and Zephyr RTOS prevents a platform from booting.
+
+The Fobnail has the potential to scale up in the future. The Fobnail architecture
+could be adapted for devices using different buses than USB. We would like to
+implement another DRTM based use case (that is out of the scope of this grant
+application) for example runtime OS attestation. The Fobnail could be used to
+check if the runtime state of the platform is trusted. It could be base on one
+of the existing projects such as Linux Kernel Runtime Guard [[11]](https://www.openwall.com/lkrg/).
+It could be useful to check the integrity of the Linux Kernel before performing
+sensitive operations.
+
+[1] "Turtles All The Way Down: Research Challenges in User-Based Attestation",
+Jonathan M. McCune, Adrian Perrig, Arvind Seshadri, Leendert van Doorn,
+https://www.usenix.org/legacy/event/hotsec07/tech/full_papers/mccune/mccune_html/index.html
+
+[2] "Remote Attestation Procedures Architecture",
+H. Birkholz, D. Thaler, M. Richardson, N. Smith, W. Pan,
+https://datatracker.ietf.org/doc/draft-ietf-rats-architecture/
+
+[3] "Fobnail Token provisioning - Fobnail architecture", M. Zmuda, N. Kamiński,
+https://drive.google.com/file/d/143wW4zWcLbedbuaWMlQpC9f_CMCcXdLH/view?usp=sharing
+
+[4] "External platform provisioning - Fobnail architecture", M. Zmuda, N. Kamiński,
```

Comment:
@Asiderr Wouldn't it be more readable if we would put images directly to let them render in a document?

Reply:
We have uploaded some PNG images but they are not used anywhere in text?

Reply:
@macpijan I'm not sure if we want to show fobnail diagrams in the project description. It's described and shown in the Fobnail architecture.

Reply:
@macpijan It is used in the [architecture.md](https://github.com/fobnail/docs/pull/1/files#diff-140eef3ba41bdcf401d507408084181f2c0ac627532b61e0f7906ea7cc926782R33)